java/dbschema/src/org/netbeans/modules/dbschema/jdbcimpl/SchemaElementImpl.java

```diff
@@ -592,18 +592,18 @@ private void initViews(ConnectionProvider cp, LinkedList views, DDLBridge bridge
                                     rset = bridge.getDriverSpecification().getRow();
                                     
                                     //test references between two schemas
-                                    c1 = (String) rset.get(new Integer(1));
-                                    s1 = (String) rset.get(new Integer(2));
-                                    c2 = (String) rset.get(new Integer(5));
-                                    s2 = (String) rset.get(new Integer(6));
+                                    c1 = rset.get(new Integer(1));
```

Comment:
Do we really need the new Integer() here?

Reply:
If we do we'll have a bigger problem. The primitive wrapper constructors will go away (they are deprecated for removal), so we need to get rid of them.

To my knowledge there is only one valid reason to use the constructors: You want an object, that you can synchronize on, that is not shared with others. But that is also flawed and will break in the future, when the primitive wrappers will be turned into value base objects, that don't have identity and don't hold a monitor.

TL;DR: `c1 = rset.get(new Integer(1));` should be `c1 = rset.get(1);`

But that might be worth fixing in another iteration.

Reply:
Ok, then let's have this in.
I've checked the code there are more than 5000 instances of new Integer, there also there are a few Long-s, Byte, Short-s and Chars as well. So that would be probably a few additional PR-s

Reply:
Just wanted to respond for completeness sake, but it would be reasonable to change like @matthiasblaesing  says. But, I didn't want to "confuse" or "cloud" the issue as I'm really trying to just reduce the warning messages here..

I'll pick that up for another change request..

BTW, thanks to @lkishalmi  for committing the change!